WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerFragment.java

```diff
@@ -679,4 +684,15 @@ private void trackSelectedOtherSourceEvents(AnalyticsTracker.Stat stat, boolean
         properties.put("is_video", isVideo);
         AnalyticsTracker.track(stat, properties);
     }
+
+    private ArrayList<EditImageData.InputData> createListOfEditImageInputData(List<Uri> uris) {
+        ArrayList<EditImageData.InputData> inputData = new ArrayList<>(uris.size());
+        for (Uri uri : uris) {
+            String stringUri = uri.toString();
+            String outputFileExtension = MimeTypeMap.getFileExtensionFromUrl(stringUri);
+            // TODO Add support for nullable lowResImgUrl
```

Comment:
I'm passing null as thumbnail URL because the same file should be used for both thumbnail and full image. However, the logic in the PreviewImageViewModel didn't work as expected when I passed the same value for both low and high res.

Reply:
Fixed here: 94ca331

Reply:
Fixed in 39074c2c389829057318343b297a3d7d364d469b